go/tools/gopackagesdriver/bazelquerydriver/driver.go

```diff
@@ -0,0 +1,323 @@
+// Copyright 2021 The Bazel Go Rules Authors. All rights reserved.
```

Comment:
Why? This is used in other files like go/private/tools/lines_sorted_test.bzl
Wouldn't "The Bazel Authors" refer to https://github.com/bazelbuild/bazel/blob/master/AUTHORS ?

Reply:
Can we add Derivita to the bazelbuild/bazel AUTHORS file as well? That would resolve my concerns.

Reply:
Re-reading the internal open source guidelines, my understanding is that "The Bazel Authors" refers to everyone who holds copyright. The AUTHORS file is supposed to be a list of significant contributors and is not supposed to have any implication for copyright ownership.

That is not what rules_go's AUTHORS file says though, and it's certainly incomplete. It seems that the advice from the open source office has changed at some point.

I've opened #2843, and I've added this to my list of things to look at when I have bandwidth for maintenance. It will take time to resolve though.

Reply:
Ok, I've also re-read the CLA. I think that makes sense.